site/_posts/2021-04-23-announcing-fusionauth-1-26.md

```diff
@@ -0,0 +1,103 @@
+
+Here's the tenant edit view, where I am configuring available MFA methods:
+
+{% include _image.liquid src="/assets/img/blogs/release-1-26/setting-mfa-options.png" alt="Configuring allowed MFA methods." class="img-fluid" figure=false %}
+
+Unfortunately, these changes are **not backwards compatible** with the previous Two Factor implementation, so if you have built on top of it, please [review the changes carefully](/docs/v1/tech/apis/two-factor/).
+
+## Entity Management
+
+When talking to customers, users and potential clients, we heard that they were often modelling entities beyond users and wanted to use FusionAuth. So, we built it; Entity Management is a premium feature requiring a paid license. It is currently in technology preview and may change; please provide feedback to us. 
+
+You can create distinct entity types representing classes of entities, such as "API" or "Company". You then can create permissions on these types, such as "invoke" or "manage". At that point, you create individual instantiations of these types, such as "Reminder API" or "Bachmanity". Finally, create "grant" relationships between entities, which allow you to model permissions between entities.
+
+For example, suppose I have a reminder API which needs to call a todo API to query upcoming tasks. With entity management, I can:
+
+* Create an "API" entity type with the following permissions: `query`, `execute`, and `configure`.
+* Create an "reminder API" entity
+* Set up a "todo API" entity
+* Grant the reminder API `query` permissions on the todo API
+
+When the reminder API is ready to gather upcoming tasks, it "signs in" with the Client Credentials grant. FusionAuth will, if the reminder API has the appropriate permission on the todo API, create a JWT. The reminder API can present the JWT, perhaps as a bearer token, to the todo API. The todo API can then process the request and return the needed data.
+
+{% include _image.liquid src="/assets/img/blogs/release-1-26/list-of-entities.png" alt="A list of APIs created using Entity Management." class="img-fluid" figure=false %}
+
+All of this set up and execution can be done via API, making Entity Management a great way to model machine to machine communication in a dynamic service oriented application, an internet of things system, or elsewhere. You don't have to use the grant mechanism either. If you want to model relationships in FusionAuth for any reason, store JSON arbitrary data in the `entity.data` field. Read more about [Entity Management here](/docs/v1/tech/core-concepts/entity-management/).
+
+## An API to manage API keys
+
+In true Inception style, there is now an API to manage FusionAuth API keys. This was a community request last year. You can create, copy, update, and delete API keys. These keys are indistinguishable from API keys created in the user interface. 
```

Comment:
These keys are indistinguishable from API keys created in the user interface. =>
These keys are indistinguishable from API keys created in the FusionAuth Admin UI interface.